code/controllers/subsystem/daynightcycle.dm

```diff
@@ -0,0 +1,77 @@
+var/datum/subsystem/daynightcycle/SSDayNight
+
+#define TOD_MORNING 	1
+#define TOD_SUNRISE 	2
+#define TOD_DAYTIME 	3
+#define TOD_AFTERNOON 	4
+#define TOD_SUNSET 		5
+#define TOD_NIGHTTIME 	6
+
+/datum/subsystem/daynightcycle
+	name          = "Day Night Cycle"
+	init_order    = SS_INIT_DAYNIGHT
+	display_order = SS_DISPLAY_DAYNIGHT
+	priority      = SS_PRIORITY_DAYNIGHT
+	wait          = 20 MINUTES
```

Comment:
This is purely opinion-based but I would actually consider putting the full day-night cycle around 40 minutes, half of that time being dark. Under your current system, sunrise (a 2 minute phenomena) lasts an equal amount of time to the night itself (a roughly 12 hour phenomena).

Reply:
Set the wait time to 10, and removed everything other than 4 times of day, made morning and night a lot darker to compensate.